### ISSUE TITLE
Confusing start and stop errors

### ISSUE THREAD

**Issue by ddobric** [Azure/iotedge#151]
Have installed IotEdge runtime (iotedge 1.0.0 (52ef77db24126bf473265fc09c53d35290a2dd6b)).
It runs well without any runtime issues.

After a while I have stopped it with Stop-Service iotedge.
Runtime stopped as expected.

Then I wanted to start it with: Start-Service iotedge
Following error occurs:

```
stop-service : Service 'iotedge (iotedge)' stop failed.
At line:1 char:1
+ stop-service iotedge
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (System.ServiceProcess.ServiceController:ServiceController) [Stop-Service], ServiceCommandException
    + FullyQualifiedErrorId : StopServiceFailed,Microsoft.PowerShell.Commands.StopServiceCommand
```

To workaround this, I switched to SCM (service control manager) and started the service manually. It is running without any issues.

Sometimes, stop-service command fails, but service is stopped.

Using Win10 x64 with Linux Docker containers.

Thanks

**Comment by avranju** [comment]
I am able to reproduce this problem. The error returned by `Stop-Service` seems to be a bogus error. I don't find anything amiss in the Windows Event logs for `iotedged` or in the Windows OS system logs. `Get-Service` seems to return the correct service status though. Do you think running `Stop-Service` like so is a reasonable workaround?
```
Stop-Service -Name iotedge -NoWait
```
This doesn't produce the error and the service stops after a few seconds.

**Comment by myagley** [comment]
This was fixed in version 1.0.4 of the IoT Edge daemon.
